securedrop/manage.py

```diff
@@ -194,6 +195,7 @@ def _add_user(is_admin=False):
                                              issuer_name='SecureDrop')
             qr = qrcode.QRCode()
             qr.add_data(uri)
+            sys.stdout = codecs.getwriter("utf-8")(sys.stdout)
```

Comment:
Could you please update the commit message to explain why this is needed ?

Reply:
sure thing, https://circleci.com/gh/freedomofpress/securedrop/3618 is an example failure otherwise.